incubator/knative/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+description: Knative Helm chart for Kubernetes
+name: knative
+version: 0.1.0
+appVersion: 0.1.1
+home: https://github.com/knative/
+icon: https://raw.githubusercontent.com/knative/docs/master/images/logo/rgb/knative-logo-rgb.jpg
+keywords:
+- knative
+- eventing
+- istio
+- serving
+- build
+sources:
+- https://github.com/knative/
+maintainers:
+- name: simplytunde
+  email: tunde@aquatribe.com
+- name: boluisa
+  email: babatunde@aquatribe.com
```

Comment:
since crd's are installed, you might want to require helm >= 2.10.0: https://github.com/helm/helm/releases

Reply:
helmVersion requirement has been added. Thanks a lot @davidkarlsen for the review